conf.yaml

```diff
@@ -89,6 +90,19 @@ contents:
               -
                 repo:   kibana
                 path:   docs
+          -
+            title:      Azure Marketplace and Resource Manager (ARM) template
+            prefix:     azure-resource-manager
```

Comment:
Likewise I think we might want to tweak this prefix so that it makes sense once there is more than just azure in the book.  Doing so now means it's less likely that we'll have to fix broken links later.

Reply:
something like `deployment/azure-resource-manager`?